### ISSUE TITLE
activate.bat fails for a path with whitespace in it on Windows

### ISSUE THREAD

**Issue by ram3ai** [conda/conda#8501]
## Current Behavior
activate.bat fails for a path with whitespace in it on Windows.
My Anaconda shortcut: `"%windir%\System32\cmd.exe" /K ""C:\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64\Scripts\activate.bat" "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64""` 
error: `\Microsoft was unexpected at this time.`

### Steps to Reproduce
Install Visual Studio with Anaconda to the default path, then update Anaconda (haven't tried to repro myself from the ground up).

### Proposed fix
```
conda/shell/Scripts/activate.bat
19c19
<     @CALL %~dp0..\condabin\conda.bat activate
---
>     @CALL "%~dp0..\condabin\conda.bat" activate
```
This fixes the issue for me. This line has been changed in https://github.com/conda/conda/commit/75ae2ca70802c6748b48ab75c9ceb27b55fb6b61

## Environment Information
<details open><summary><code>`conda info`</code></summary><p>

```
     active environment : base
    active env location : C:\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64
            shell level : 1
       user config file : C:\Users\evgre\.condarc
 populated config files : C:\Users\evgre\.condarc
          conda version : 4.6.9
    conda-build version : 3.10.5
         python version : 3.6.5.final.0
       base environment : C:\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/win-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64\pkgs
                          C:\Users\evgre\.conda\pkgs
                          C:\Users\evgre\AppData\Local\conda\conda\pkgs
       envs directories : C:\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64\envs
                          C:\Users\evgre\.conda\envs
                          C:\Users\evgre\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.6.9 requests/2.18.4 CPython/3.6.5 Windows/10 Windows/10.0.17763
          administrator : False
             netrc file : None
           offline mode : False
```
</p></details>

**Comment by msarahan** [comment]
please try conda from the conda-canary channel: ``conda update conda -c conda-canary``

I think I might have fixed this in 4.6.10

**Comment by msarahan** [comment]
never mind, Ray's change will fix it.
